drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -5634,7 +5634,7 @@ void RasterizerStorageGLES2::initialize() {
 	config.multisample_supported = config.extensions.has("GL_EXT_framebuffer_multisample") || config.extensions.has("GL_EXT_multisampled_render_to_texture") || config.extensions.has("GL_APPLE_framebuffer_multisample");
 
 #ifdef GLES_OVER_GL
-	config.use_rgba_2d_shadows = false;
+	config.use_rgba_2d_shadows = true;
```

Comment:
This should remain false because desktop always supports floating point textures. Floating point textures are preferred, encoding to RGBA is only used as a fallback when floating point isnt available.